compiler/dyno/lib/queries/ErrorMessage.cpp

```diff
@@ -82,6 +82,7 @@ void ErrorMessage::addDetail(ErrorMessage err) {
 }
 
 void ErrorMessage::swap(ErrorMessage& other) {
+  (void)id_;                    // mark field as used for linter
```

Comment:
I think this should be swapped just before `message_.swap`. Did I miss/forget something?